tests/upgrade/0.8.2-0.9.0/jiva/volume_test.go

```diff
@@ -72,6 +78,39 @@ var _ = Describe("[jiva] TEST VOLUME UPGRADE", func() {
 			podCount := ops.GetPodRunningCountEventually("default", upgradedLabel, replicaCount+1)
 			Expect(podCount).To(Equal(replicaCount+1), "while checking pod version")
 
+			By("verifying registered replica count and replication factor")
+			podList, err := ops.PodClient.List(metav1.ListOptions{LabelSelector: ctrlLabel})
+			Expect(err).ShouldNot(HaveOccurred(), "while listing controller pod")
+
+			replicationFactor := ""
+			for _, env := range podList.Items[0].Spec.Containers[0].Env {
+				if env.Name == "REPLICATION_FACTOR" {
+					replicationFactor = env.Value
+				}
+			}
+			Expect(replicationFactor).ToNot(Equal(""), "while fetching replication factor")
+
+			curl := "curl http://localhost:9501/v1/volumes"
+			cmd := []string{"/bin/bash", "-c", curl}
+			opts := tests.NewOptions().
+				WithPodName(podList.Items[0].Name).
+				WithNamespace(podList.Items[0].Namespace).
+				WithContainer(podList.Items[0].Spec.Containers[0].Name).
+				WithCommand(cmd...)
+
+			out, err := ops.ExecPod(opts)
```

Comment:
Please discuss with @sagarkrsd on moving this logic from tests.ops to `pkg/kubernetes/pod` & `pkg/kubernetes/podexec`

Subsequent PRs can refactor this particular pod exec section

Reply:
Sagar is still working on it. Once he is done with it will update the BDD.

Reply:
Thanks @shubham14bajpai , @AmitKumarDas Once refactoring PR gets merged, it can be refactored.